### ISSUE TITLE
Clicking beer is inaccessible for the blind?

### ISSUE THREAD

**Issue by blindsoose** [wol-soft/beerplop#71]
Hello, so as you have guessed, i'm a blind person, and when i'm launching this game, i obviously try to play it by finding something to click on.
At first, i thought you need to click on plops: 0, but no, this opens the owned buildings/plop production thingy.
Then i've tried meny more graphics to click on, for example a+, the X labeled graphics, bar graphics and so on, but my plops are not increasing.
So, i guess beer click button is unlabeled? if so, is it possible to label it so blinds also can play this game?
This game seems very interesting to me, and i'd be happy to play it.

**Comment by wol-soft** [comment]
Hi @blindsoose do you use any tools to access Beerplop so I can comprehend your issue?

**Comment by blindsoose** [comment]
only nvda from
http://nvaccess.org

**Comment by wol-soft** [comment]
I've added a label with the version 1.61.3.

Can you check if it helps? I guess there will occur a lot of problems using Beerplop with nvda.

**Comment by blindsoose** [comment]
Hello, yes, it works. Thank you very much.
And i guess another problem that's left is only the hover things to get building details. It's doable with nvda, but very uncomfortable.
Thanks for your hard work.

**Comment by wol-soft** [comment]
Hi @blindsoose,
I've added labels to the building details. If further problems occur feel free to open a new issue/reopen this one.
Thanks for your input!

**Comment by blindsoose** [comment]
Hello, i'm sorry, i may have requested a wrong feature. When i said building details, i ment the thing when you hover over a building name and it will tell you this:
Openers
Openers to open bottles
Owned:
100
Total production:
299,001 Million Plops
Production:
5 494,8 Plops (3,0% of total plops)
Production each:
49,5 Plops.
Can you please make so that it will be possible to don't hover it, but to just press on some button and a dialog with those stats will pop up? O r just put those details in price section?
Also, is there a thing to know what a level up will give and how much it will cost?
Also, i have the number 50 if i will press down arrow after the Max button. What is this?
Thanks for your hard work again, and sorry for troubling you.

**Comment by wol-soft** [comment]
Hi @blindsoose,

I've added the information from the popover to the building detail modal which opens when you click on the building (same spot where the hover popover currently opens). Added a label to the buildings, eg. "Building details for openers".

Level Ups always duplicate the production of a building (as well the normal buildings as the Bottle Cap Factories).

The 50 is a custom buy charge which can be configured with a modal opening via the button next to the 50 containing a gear.

Released the changes with version 1.65.0. Hope they help you.
